src/_packagedcode/pypi.py

```diff
@@ -684,10 +696,44 @@ def parse(cls, location):
 
         metadata = {}
         parser = ConfigParser()
+        dependent_packages = []
         with open(location) as f:
             parser.read_file(f)
-
         for section in parser.values():
+            if section.name == 'options':
+                scope_by_sub_section = {
+                    "install_requires": "install",
+                    "tests_require": "test",
+                    "setup_requires": "setup",
+                    "python_requires": "python",
+                }
+                for sub_section in scope_by_sub_section:
+                    if sub_section not in section:
+                        continue
+                    scope = scope_by_sub_section[sub_section]
+                    if scope != "python":
+                        reqs = list(get_requirement_from_section(section=section, sub_section=sub_section))
+                        dependent_packages.extend(cls.parse_reqs(reqs, scope))
+                        continue
+                    python_requires = section[sub_section]
+                    purl = PackageURL(
+                        name="python",
+                        type="generic"
+                    )
+                    resolved_purl = is_purl_resolved(purl = purl, specifiers= SpecifierSet(python_requires))
+                    dependent_packages.append(models.DependentPackage(
+                    purl=str(resolved_purl.purl),
+                    scope=scope,
+                    is_runtime=True,
+                    is_optional=False,
+                    is_resolved=resolved_purl.is_resolved,
+                    extracted_requirement=f"python_requires{python_requires}",
```

Comment:
I am not sure that's what you want here.

Reply:
I have made a test here https://github.com/nexB/python-inspector/pull/31/files#diff-ea501ea9847f41dae13d2abfce71d6818cae1c0873363f295ce4904d79c90571 which I think gives expected output python_requires>=3.6.0,<3.8.0, I have changed the name of variable which I think makes it more clear now.